slamon_afm/tests/models_tests.py

```diff
@@ -95,15 +97,50 @@ def testClaimUnassignedTask(self):
         claimed = list(Task.claim_tasks(self.agent, 1))
         self.assertEqual(len(claimed), 1)
 
+    def testClaimTaskOrder(self):
+        """
+        Ensure the oldest task is always served first
+        """
+        times = list(permutations([
+            datetime(2015, 10, 1, 10, 0, 0),
+            datetime(2015, 10, 1, 11, 0, 0),
+            datetime(2015, 10, 1, 12, 0, 0)
+        ]))
+
+        def create_uuid(*args):
+            return str(uuid5(UUID('de305d54-75b4-431b-adb2-eb6b9e546013'), '.'.join(map(str, args))))
+
+        for seq in range(len(times)):
```

Comment:
`xrange` is preferred in for-loops: `xrange` returns generator and `range` returns a list...